src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{std::move(ca), std::move(ref)}
```

Comment:
I agree that `std::move(ca)` is not necessary. But other changes need for success compilation under mingw